### PR TITLE
NL Blog CAPP12 klantenpanel

### PR DESCRIPTION
**Pull request by SusanDefacto**
still working on pictures

### CODE REVIEW

File: source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Defacto presenteert nieuwe versie van CAPP"
+date: 2017-12-06 8:00 CEST
+tags: nieuws, Defacto
```
Comment (fatboypunk) [inline source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown]: I think `nieuws` should be with a capital `Nieuws` :)

Comment (RonaldDijkstra) [reply source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown]: No, unless the other blogs with nieuws get Nieuws as well! ;-)

Comment (fatboypunk) [reply source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown]: Ah confusing since the tag on the website does it with a capital

Comment (RonaldDijkstra) [reply source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown]: Very true. Will think about it.

**Comment by RonaldDijkstra** [comment]
> - ./build/index.html
  *  internally linking to /blog/defacto-presenteert-nieuwe-CAPP/, which does not exist (line 24)

Which does not exist yet!

File: source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+author: Susan
+lang: nl
+featured: true
+image: images/blog/social/klantenpanel-1200x630.jpg
+featured_image: images/blog/featured/20171211-klantendag.jpg
+---
+
+Afgelopen donderdag werd er in Slot Doddendeal bij Nijmegen het eerste van de in totaal vier klantenpanels gehouden. De focus lag op de nieuwe versie van CAPP: CAPP 12.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown]: Niet afgelopen donderdag meer

File: source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Defacto presenteert nieuwe versie van CAPP"
+date: 2017-12-06 8:00 CEST
```
Comment (RonaldDijkstra) [inline source/blog/nl/2017-12-06-defacto-presenteert-nieuwe-capp-versie.html.markdown]: Update datum (ook in filename)

**Comment by fatboypunk** [comment]
The deletions of the images is oke? Maybe it's better to do those things in seperate PR's in the future.

**Comment by RonaldDijkstra** [comment]
You're right @fatboypunk !

**Pull request merged**